README.md

```diff
@@ -42,6 +42,11 @@ If you are using Maven without Bom, Add this to your dependencies.
 ```
 If you are using Gradle, add this to your dependencies
 ```Groovy
+repositories {
+        google() //this is required from version 1.104 onwards
```

Comment:
comment can be removed. This won't be necessary in the near future.

Reply:
Sure. (when I make this change.. is it okay to amend and force push or a separate commit)?

Reply:
@elharo done !!

Reply:
Just make multiple commits and push again. I'll squash when it's merged.

Reply:
@elharo done